webpack/assets/javascripts/react_app/redux/actions/factCharts/index.js

```diff
@@ -0,0 +1,29 @@
+import {
+  FACT_CHART_DATA_REQUEST,
+  FACT_CHART_DATA_SUCCESS,
+  FACT_CHART_DATA_FAILURE,
+  OPEN_MODAL,
```

Comment:
Action names are global - OPEN/CLOSE are too generic names, it should be specific to the page / action